docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+db:
+  image: mdillon/postgis:9.4
```

Comment:
I think that's a good solution; this image is permissively licensed so it would be easy to fork and make our own. I'll leave this as-is for now until we've got the account set up.